PathPlanning/PotentialFieldPlanning/potential_field_planning.py

```diff
@@ -106,14 +106,17 @@ def potential_field_planning(sx, sy, gx, gy, ox, oy, reso, rr):
 
     rx, ry = [sx], [sy]
     motion = get_motion_model()
+    previous_id = [(None, None)] * 3
```

Comment:
Maybe the variable name should be "xy_history" or something like that. And number 3 should be global parameters like "AREA_WIDTH".

Reply:
added